04-manpages.patch by Anibal Monsalve Salazar <anibal@debian.org>

Fixed a number of typos in man pages, closes: #305653, #305654, #305656, #305657, #305658
Fixed "manpage fails to escape hyphen", closes: #201034.

Signed-off-by: Anibal Monsalve Salazar <anibal@debian.org>

--- a/floppyd.1	2009-11-04 07:34:01.000000000 +1100
+++ b/floppyd.1	2010-02-15 12:37:35.000000000 +1100
@@ -1,6 +1,7 @@
 .TH floppyd 1 "03Nov09" mtools-4.0.12
 .SH Name
 floppyd - floppy daemon for remote access to floppy drive
+.br
 floppyd_installtest - tests whether floppyd is installed and running
 '\" t
 .de TQ
--- a/mattrib.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mattrib.1	2010-02-15 12:37:35.000000000 +1100
@@ -56,7 +56,7 @@ Hidden bit.  Used to make files hidden f
 Recursive.  Recursively list the attributes of the files in the subdirectories.
 .TP
 \&\fR\&\f(CWX\fR\ 
-Concise. Prints the attributes whithout any whitespace padding.  If
+Concise. Prints the attributes without any whitespace padding.  If
 neither the "/" option is given, nor the \fImsdosfile\fR contains a
 wildcard, and there is only one Msdos file parameter on the command
 line, only the attribute is printed, and not the filename.  This option
--- a/mdel.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mdel.1	2010-02-15 12:37:35.000000000 +1100
@@ -1,6 +1,7 @@
 .TH mdel 1 "03Nov09" mtools-4.0.12
 .SH Name
 mdel - delete an MSDOS file
+.br
 mdeltree - recursively delete an MSDOS directory and its contents
 '\" t
 .de TQ
--- a/mdir.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mdir.1	2010-02-15 12:37:35.000000000 +1100
@@ -49,7 +49,7 @@ Fast.  Do not try to find out free space
 the amount of free space takes up some non trivial amount of time, as
 the whole FAT must be read in and scanned.  The \fR\&\f(CW-f\fR flag bypasses
 this step.  This flag is not needed on FAT32 filesystems, which store
-the size explicitely.
+the size explicitly.
 .TP
 \&\fR\&\f(CWb\fR\ 
 Concise listing. Lists each directory name or filename, one per line
--- a/mpartition.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mpartition.1	2010-02-15 12:37:35.000000000 +1100
@@ -66,7 +66,7 @@ Creates the partition described by \fIdr
 can be bootable at a time.
 .TP
 \&\fR\&\f(CWd\fR\ 
-"Desactivates" the partition, i.e. makes it unbootable.
+"Deactivates" the partition, i.e. makes it unbootable.
 .PP
 If no operation is given, the current settings are printed.
 .PP
@@ -104,7 +104,7 @@ partition table:
 .TP
 \&\fR\&\f(CWf\fR\ 
 Usually, before writing back any changes to the partition, mpartition
-performs certain consistenct checks, such as checking for overlaps and
+performs certain consistency checks, such as checking for overlaps and
 proper alignment of the partitions.  If any of these checks fails, the
 partition table is not changes.  The \fR\&\f(CW-f\fR allows you to override
 these safeguards.
--- a/mtools.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mtools.1	2010-02-15 12:37:35.000000000 +1100
@@ -444,7 +444,7 @@ packed so close together that we need to
 doesn't exist for 20 sector formats.
 .PP
 These formats are supported by numerous DOS shareware utilities such as
-\&\fR\&\f(CWfdformat\fR and \fR\&\f(CWvgacopy\fR. In his infinite hybris, Bill Gate$
+\&\fR\&\f(CWfdformat\fR and \fR\&\f(CWvgacopy\fR. In his infinite hubris, Bill Gate$
 believed that he invented this, and called it \fR\&\f(CW\(ifDMF disks\(is\fR, or
 \&\fR\&\f(CW\(ifWindows formatted disks\(is\fR. But in reality, it has already existed
 years before! Mtools supports these formats on Linux, on SunOs and on
@@ -470,7 +470,7 @@ Mtools supports these formats only on Li
 The 2m format was originally invented by Ciriaco Garcia de Celis. It
 also uses bigger sectors than usual in order to fit more data on the
 disk.  However, it uses the standard format (18 sectors of 512 bytes
-each) on the first cylinder, in order to make these disks easyer to
+each) on the first cylinder, in order to make these disks easier to
 handle by DOS. Indeed this method allows to have a standard sized
 bootsector, which contains a description of how the rest of the disk
 should be read.
